src/FullBoard.cpp

```diff
@@ -131,7 +131,7 @@ int FullBoard::update_board(const int color, const int i) {
     auto eyeplay = (m_neighbours[i] & s_eyemask[!color]);
 
     auto captured_stones = 0;
-    int captured_sq;
+    auto captured_sq = -1;
```

Comment:
I've rejected this before and I'll keep rejecting it. If the warning bothers you, change the build flags to disable it. I'll accept patches for that.

But we don't make our code worse because one particular compiler is stupid.

Reply:
Sounds good, also disabled the cl2.hpp warning that was bothering me. :)